packages/apt/test.sh

```diff
@@ -251,23 +220,31 @@ echo "::endgroup::"
 
 
 echo "::group::Upgrade test"
-if [ -n "${old_package}" ]; then
-  sudo apt purge -V -y \
-    ${package} \
-    "${mysql_package_prefix}-*"
-  sudo rm -rf /var/lib/mysql
-
-  sudo mv /etc/apt/sources.list.d/${package}.list /tmp/
-  case ${package} in
-    mysql-community-8.*)
-      mysql_community_install_mysql_apt_config
-      ;;
-  esac
-  sudo apt update
-  sudo apt install -V -y ${old_package}
+sudo apt purge -V -y \
+  ${package} \
+  "${mysql_package_prefix}-*"
+sudo rm -rf /var/lib/mysql
+
+sudo mv /etc/apt/sources.list.d/${package}.list /tmp/
+sudo apt update
+case ${package} in
+  mysql-community-8.*)
+    mysql_community_install_mysql_apt_config
+    ;;
+esac
+
+if [[ "${package}" == mariadb-* && "${distribution}-${code_name}" == "debian-bookworm" ]]; then
+  # TODO: Remove debian-bookworm after we release a new
+  # version. We can't do upgrade test because the previous Mroonga
+  # requires old MariaDB that isn't provided by Debian.
+  echo "Skipping upgrade test about mariadb package for Debian Bookworm."
```

Comment:
Is this needed?

Reply:
`mariadb-10.11-mroonga` が依存している `MariaDB`のバージョンが古い問題がまだ存在していると判断して、
それがわかるように分岐させていました。
```
  + apt show mariadb-10.11-mroonga
  + sudo apt install -V -y mariadb-10.11-mroonga
  
  WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
  
  Reading package lists...
  Building dependency tree...
  Reading state information...
  Some packages could not be installed. This may mean that you have
  requested an impossible situation or if you are using the unstable
  distribution that some required packages have not yet been created
  or been moved out of Incoming.
  The following information may help to resolve the situation:
  
  The following packages have unmet dependencies:
   mariadb-10.11-mroonga : Depends: mariadb-server (= 1:10.11.3-1) but it is not going to be installed
                           Depends: mariadb-server-core (= 1:10.11.3-1) but it is not going to be installed
  E: Unable to correct problems, you have held broken packages.
```
https://github.com/otegami/mroonga/actions/runs/9460769179/job/26060246435#step:17:4376

Reply:
ref: Debian Bookworm MariaDB v1:10.11.6-0+deb12u1
- https://packages.debian.org/source/bookworm/mariadb

Reply:
だったらこれは失敗するままでいいです。
リリースしたら解決するんですよね？

Reply:
下記の理解で解決すると思っているので、一旦この変更は消して失敗する状態に戻します。
古い MariaDB に依存した Mroonga をアップグレードテストで参照しなくなるので、
リリースしたら解決すると理解しています。